metrics/mobile-app-metrics.md

```diff
@@ -0,0 +1,108 @@
+# Abstract
+
+This document contains the proposal for gathering metrics on system statistics for running Mobile Apps and making them available to developers.
+
+## Goals
+
+### Primary Goals
+
+- Provide mobile app developers a service that can receive overall runtime metrics (processor and memory data, authentication events, runtime versions, networking events, etc).
```

Comment:
@david-martin @wtrocki @pb82 
Should we generalize things? Or just talk about mobile app version and sdk version metrics?

Reply:
Maybe you can leave the generalization but mention that initially we will only record SDK and APP version?